src/Microsoft.XmlSerializer.Generator/src/Microsoft.XmlSerializer.Generator.csproj

```diff
@@ -0,0 +1,90 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
This looks like it is trying to use the new CLI tools which currently corefx doesn't support using.